about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>KELSEY MCALPINE | About</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles/styles.css">
+  </head>
+  <body id="about">
+    <header>
+      <section id="header-top">
+        <h5 class="left">FULL-STACK DEVELOPER | SEATTLE, WA</h5>
+        <ul class="right">
+          <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+          <li><a href="#"><i class="fa fa-github"></i></a></li>
+        </ul>
+      </section>
+      <section id="header-bottom">
+        <h1 class="left">KELSEY MCALPINE</h1>
```

Comment:
This is a personal preference. I would make this a link to bring the user back to the Homepage (index).